### DISCUSSION TITLE
Which subscription is better?

### DISCUSSION THREAD

**Discussion by preetjhotavoni** [community/community#28156] [New to GitHub] [answered]
In our organization, we have 20 members. We just want to sync and save the codes in private. Which is the best subscription?

**Comment by jfullstackdev** [comment, marked as answer]
make a repo, add them, add your private project, no worries it's free

**Comment by preetjhotavoni** [comment]
What is the maximum storage available for the individual private repository?

**Reply by mazkdevf** [reply]
Probably 500 MB OR 1 GB
